readthedocs/settings/base.py

```diff
@@ -848,6 +848,8 @@ def SOCIALACCOUNT_PROVIDERS(self):
     # we can't make use of the DJSTRIPE_SUBSCRIBER_MODEL setting.
     DJSTRIPE_SUBSCRIBER_CUSTOMER_KEY = None
 
+    SLACK_WEBHOOK_SALES_CHANNEL = None
```

Comment:
This should probably have a comment explaining what we expect here? Is it a string of the channel name, or some hashed thing?